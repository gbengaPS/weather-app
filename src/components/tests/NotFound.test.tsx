import React from 'react';
import { render } from '@testing-library/react';
import NotFound from '../NotFound';


describe('NotFound', () => {
  it('should match snapshot', () => {
    const { container } = render(<NotFound />);
    expect(container).toMatchSnapshot();
  });
});
