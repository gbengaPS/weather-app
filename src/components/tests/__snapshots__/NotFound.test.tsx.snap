// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match snapshot 1`] = `
<div>
  <h1>
    Not Found
  </h1>
</div>
`;
